cordyceps/rust-toolchain.toml

```diff
@@ -0,0 +1,10 @@
+[toolchain]
```

Comment:
So, this is allowed on the current stable and nightly, but not on the project level nightly. Added a file here in case you don't want to rev the whole project.

This is due to the const bounds on the const constructor method. I can also feature flag that out for easier compat.

Reply:
happy to bump nightly if nothing’s broken (and if anything is broken, still happy to bump nightly but i’d have to fix that first)

Reply:
Pushed a commit updating the main toolchain version. I'm not sure if there is still the existing cordyceps failures we talked about present.